store/tikv/region_cache.go

```diff
@@ -16,22 +16,35 @@ package tikv
 import (
 	"bytes"
 	"sync"
+	"time"
+	"sync/atomic"
 
-	log "github.com/Sirupsen/logrus"
 	"github.com/juju/errors"
+	log "github.com/Sirupsen/logrus"
 	"github.com/petar/GoLLRB/llrb"
 	"github.com/pingcap/kvproto/pkg/kvrpcpb"
 	"github.com/pingcap/kvproto/pkg/metapb"
 	"github.com/pingcap/pd/pd-client"
 	goctx "golang.org/x/net/context"
 )
 
+const (
+	rcDefaultRegionCacheTTL	= time.Minute * 2
+)
+
+// CachedRegion encapsulates {Region, TTL}
+type CachedRegion struct {
+	region	*Region
+	ttl		int64
```

Comment:
`ttl` is a duration, not a timestamp, why not name it `lastAccess` directly?

Reply:
Ok, I will change it. Thanks~